zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I have no idea why its uppercase now. I did not touch this line. Maybe the editor (I am using notpad++)? Or the style formatter?